### PR TITLE
Enable Helm charts

### PR DESCRIPTION
**Pull request by dwaynebailey**
https://tools.hmcts.net/jira/browse/RDM-3907

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/ccd-definition-designer-api/values.template.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ELASTIC_SEARCH_PASSWORD: ''
+    ELASTIC_SEARCH_PORT: 9200
+    ELASTIC_SEARCH_SCHEME: http
+    ELASTIC_SEARCH_ENABLED: false
+    ELASTIC_SEARCH_INDEX_SHARDS: 2
+    ELASTIC_SEARCH_INDEX_SHARDS_REPLICAS: 1
+    ELASTIC_SEARCH_FAIL_ON_IMPORT: false
+    ELASTIC_SEARCH_DYNAMIC: false
+    ELASTIC_SEARCH_CASE_INDEX_NAME_FORMAT: '%s_cases'
+
```
Comment (adr1ancho) [inline charts/ccd-definition-designer-api/values.template.yaml:42]: Are you sure we need ELASTIC_SEARCH_* defined?

File: charts/ccd-definition-designer-api/values.aat.template.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+java:
+  environment:
+    IDAM_USER_URL: https://preprod-idamapi.reform.hmcts.net:3511
+    IDAM_S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+    USER_PROFILE_HOST: http://ccd-user-profile-api-aat.service.core-compute-aat.internal
+    AZURE_STORAGE_DEFINITION_UPLOAD_ENABLED: false
```
Comment (adr1ancho) [inline charts/ccd-definition-designer-api/values.aat.template.yaml:6]: I don't think `AZURE_STORAGE_DEFINITION_UPLOAD_ENABLED` and `USER_PROFILE_HOST` are required any more

File: charts/ccd-definition-designer-api/values.template.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    DEFINITION_DESIGNER_DB_HOST: ${SERVICE_NAME}-postgres
+    DEFINITION_DESIGNER_DB_PORT: 5432
+    DEFINITION_DESIGNER_DB_NAME: ccd_definition_designer
+    DEFINITION_DESIGNER_DB_USERNAME: ccd
+    DEFINITION_DESIGNER_DB_PASSWORD: ccd
+    DEFINITION_DESIGNER_DB_USE_SSL: false
+    ENABLE_DB_MIGRATE: true
+
+    IDAM_USER_URL: http://betaDevBccidamAppLB.reform.hmcts.net
+    DEFINITION_STORE_IDAM_KEY: ${CCD_DEFINITION_SERVICE_SECRET}
```
Comment (adr1ancho) [inline charts/ccd-definition-designer-api/values.template.yaml:24]: I need to check the env variable name for the key.  Looking at ccd-docker, should it be

      DEFINITION_DESIGNER_IDAM_KEY: ${CCD_DEFINITION_SERVICE_SECRET}

File: charts/ccd-definition-designer-api/values.template.yaml (hunk 1 above)
Comment (dwaynebailey) [reply charts/ccd-definition-designer-api/values.template.yaml:42]: Probably not, but as long as they're present in `infrastructure/main.tf` I add them to the chart

File: charts/ccd-definition-designer-api/values.template.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    DEFINITION_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_admin,jui_webapp,pui_webapp
```
Comment (adr1ancho) [inline charts/ccd-definition-designer-api/values.template.yaml:26]: I remember I changed this string.  Let me check 

Looking at ccd-docker it looks like we need only

      DEFINITION_DESIGNER_S2S_AUTHORISED_SERVICES: ccd_gw,ccd_admin

File: charts/ccd-definition-designer-api/values.template.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+    #AZURE_STORAGE_CONNECTION_STRING = "${data.azurerm_key_vault_secret.storageaccount_primary_connection_string.value}"
+    #AZURE_STORAGE_BLOB_CONTAINER_REFERENCE = "${azurerm_storage_container.imports_container.name}"
+    AZURE_STORAGE_DEFINITION_UPLOAD_ENABLED: true
```
Comment (adr1ancho) [inline charts/ccd-definition-designer-api/values.template.yaml:30]: We do not need `AZURE_STORAGE_DEFINITION_UPLOAD_ENABLED`  I believe

File: charts/ccd-definition-designer-api/values.aat.template.yaml (hunk 2 above)
Comment (dwaynebailey) [reply charts/ccd-definition-designer-api/values.aat.template.yaml:6]: Again yes, but not really a concern for Helm more a concern for the code cleanup for the repo.  And not a task for this PR

File: charts/ccd-definition-designer-api/values.template.yaml (hunk 3 above)
Comment (dwaynebailey) [reply charts/ccd-definition-designer-api/values.template.yaml:24]: Most likely, not sure if you need it.

File: charts/ccd-definition-designer-api/values.template.yaml (hunk 5 above)
Comment (dwaynebailey) [reply charts/ccd-definition-designer-api/values.template.yaml:30]: Probably not, but again it needs to be removed from all the infrastructure then.

File: charts/ccd-definition-designer-api/values.template.yaml (hunk 4 above)
Comment (dwaynebailey) [reply charts/ccd-definition-designer-api/values.template.yaml:26]: Indeed, adjusted.

**Review by adr1ancho — APPROVED** [review]
This pr is to enable helm and another pr is to get helm working

**Pull request merged**